mmcv/image/io.py

```diff
@@ -131,10 +131,17 @@ def imread(img_or_path, flag='color', channel_order='bgr'):
             candidates are `color`, `grayscale` and `unchanged`.
             Note that the `turbojpeg` backened does not support `unchanged`.
         channel_order (str): Order of channel, candidates are `bgr` and `rgb`.
+        backend (str|None): The image decoding backend type. Options are `cv2`,
+            `pillow`, `turbojpeg`, `None`. If backend is None, the global
+            imread_backend will be used.
```

Comment:
Change the last line to:
If backend is None, the global imread_backend specified by \`mmcv.use_backend\` will be used.